app/src/main/java/org/p2p/wallet/auth/repository/FileRepository.kt

```diff
@@ -91,42 +91,48 @@ class FileRepository(
                 )
                 val resolver = context.contentResolver
                 saveBitmapToFile(mainDir, fileName, bitmap).also { savedFile ->
-                    resolver.openInputStream(Uri.fromFile(savedFile)).use { input ->
-                        val contentValues = ContentValues().apply {
-                            put(MediaStore.Images.Media.DISPLAY_NAME, "$fileName.png")
-                            put(MediaStore.Images.Media.MIME_TYPE, mimeType)
-                            put(MediaStore.Images.Media.RELATIVE_PATH, "${Environment.DIRECTORY_PICTURES}/$appName")
-                            put(MediaStore.Images.Media.IS_PENDING, 1)
-                        }
+                    if (toPublic) {
+                        resolver.openInputStream(Uri.fromFile(savedFile)).use { input ->
+                            val contentValues = ContentValues().apply {
```

Comment:
https://android.github.io/android-ktx/core-ktx/androidx.content/content-values-of.html